app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/worker/DownloadKeysWorker.kt

```diff
@@ -23,15 +20,17 @@ class DownloadKeysWorker(
         const val TAG = "DOWNLOAD_KEYS"
     }
 
-
     private val exposureNotificationsRepository: ExposureNotificationsRepository by inject()
     private val serverRepository: ExposureServerRepository by inject()
+    private val analytics = FirebaseAnalytics.getInstance(context)
 
     override suspend fun doWork(): Result {
+        analytics.logEvent("key_export_download_started", Bundle())
```

Comment:
tyto analytics klíče bych spíše vložil jako `translatable="false"` do nějakého `analytics.xml` resource

Reply:
Chápu přesunutí všech klíčů do jednoho souboru, ale v XML mi to nedává smysl, není k tomu potřeba přistupovat z XML a není to potřeba měnit pro flavory. Dám do nějakého Kotlin objektu s konstantami?

Reply:
ok, tak teda jako konstanty

Reply:
done